SwifterDemoiOS/AppDelegate.swift

```diff
@@ -32,8 +32,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
-        Swifter.handleOpenURL(url)
-        return true
+        return Swifter.application(app, open: url, options: options)
```

Comment:
As above, don't change the method signature and don't pass in any platform-specific types into Swifter unless we specifically want to guard against it.

Reply:
Fixed!
https://github.com/mattdonnelly/Swifter/pull/271/commits/ba7979548be09053fd5764dbde9fd1e07c795b18